### DISCUSSION TITLE
Where do older releases get stored?

### DISCUSSION THREAD

**Discussion by bruno-rescale** [OpenRadioss/OpenRadioss#2108] [Q&A]
Hi, I'm aware the openradioss releases are published here https://github.com/OpenRadioss/OpenRadioss/releases

As I can see, due to the fast development, I notice for instance if start with 2023-11-14, the source tarball may disappear from the website in a few weeks. If i want to reference this 2023-11-14 or any other dated tarball; do these get stored somewhere else? If so, what's the website/location?

**Comment by OlivierWienholtz** [comment]
Hello,

We keep the last build of each day in the release area. 
After this the releases are deleted.

If you need a previous / specific release you can always rebuild it by checking out to previous CommitID

Olivier

**Reply by bruno-rescale** [reply]
Thanks for your kind response @OlivierWienholtz. 

So I did store the source tarball for 2023-11-14 on my computer for when i need to install it; but if I didn't do this, I see it's no longer on the release site today (last build of this post is 11-17). For future reference, is there documentation on how to rebuild with a previous CommitID? i.e. how do i know which commit belongs to 11-14?

**Reply by PaulAltair** [reply]
Hi, you may see reply to similar question here: https://github.com/OpenRadioss/OpenRadioss/discussions/2044#discussioncomment-7523894

**Comment by OlivierWienholtz** [comment]
Hello, 
In addition to Paul,
consider to clone the OpenRadioss repository with Git.
(Take care, OpenRadioss is using LFS).

You can then find the commitID (Sha1) either on GitHub  / Commits side, 
or using 'git log' command on your local machine.

then git checkout [full git Sha1] as discribes in discution #2044
